Vector/Model/RoomList/RecentsDataSource.m

```diff
@@ -110,12 +109,23 @@ - (void)dataSource:(MXKDataSource*)dataSource didStateChange:(MXKDataSourceState
     }
 }
 
+- (void)refreshRoomsSectionsAndReload
+{
+    if (!self.movingCellIndexPath)
```

Comment:
Could you please add a comment here.
"Refresh is disabled during drag&drop animation"